src/inputs/labeled-inputs.scss

```diff
@@ -52,26 +53,36 @@
     font-size: $iui-font-size-small;
     line-height: $iui-line-height;
     margin-top: $iui-inputs-margin;
-
     @include themed {
       color: t(iui-text-color-muted);
     }
 
-    svg {
+    > svg,
+    :where(:not(.iui-button)) svg {
       width: $iui-icons-default;
       height: $iui-icons-default;
       margin-right: $iui-xs;
-
       @include themed {
         fill: t(iui-icons-color);
       }
     }
 
+    > .iui-borderless {
+      border-top-left-radius: 0;
+      border-bottom-left-radius: 0;
+    }
+
     a {
       user-select: none;
     }
   }
 
+  &:not(.iui-inline) > .iui-message > .iui-borderless {
+    position: absolute;
+    top: $iui-l + 1;
```

Comment:
Try centering like inline:
![image](https://user-images.githubusercontent.com/36186912/116847858-1bb2ba80-abf4-11eb-8e16-4351c09820d5.png)

Reply:
Good suggestion, implemented.

Reply:
It won't work for non-inline
![image](https://user-images.githubusercontent.com/9084735/116939208-f5177280-ac39-11eb-92da-234eeaab4ab7.png)

But we've removed the non-inline example from html (for now) so maybe remove it here from css too

Reply:
Maybe yeah, remove it for now and finish it up later with input + button combo.

Reply:
Reverted back to `top: $iui-l + 1;`.